src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/page.tsx

```diff
@@ -162,8 +178,8 @@ export default async function DashboardPage() {
       userScanData={latestScan}
       userBreaches={subBreaches}
       enabledFeatureFlags={enabledFeatureFlags}
-      monthlySubscriptionUrl={monthlySubscriptionUrl}
-      yearlySubscriptionUrl={yearlySubscriptionUrl}
+      monthlySubscriptionUrl={`${monthlySubscriptionUrl}&${additionalSubplatParams.toString()}`}
+      yearlySubscriptionUrl={`${yearlySubscriptionUrl}&&${additionalSubplatParams.toString()}`}
```

Comment:
Q: Could this result in any duplicated query params, like potentially conflicting `utm_*` parameters?

~And is the `yearlySubscriptionUrl` (and monthly) always guaranteed to have a `?`?~
Ah, I think it does, per https://github.com/mozilla/blurts-server/blob/0b4b93bfca8acb6a3f8df03013ecd0bf25e5c14c/src/app/functions/server/getPremiumSubscriptionUrl.ts#L32